### PR TITLE
Issue 1073

### PR DESCRIPTION
**Pull request by richelleyy**
I didn't notice that Larissa added more specs for body text.

### CODE REVIEW

File: styles/books/chemistry/book.scss [hunk 1]
```diff
@@ -6,6 +6,7 @@
 
 
 //@use
+@include use('Preface','carnival:::Preface');
 @include use('KeyTerm','carnival:::EocKeyTerm');
 @include use('KeyEquation','carnival:::EocKeyEqu');
```
Comment (helenemccarron) [inline styles/books/chemistry/book.scss]: I'm not thrilled with shortening names, it feels like EocKeyEquations would be more readable but that's nitpicking

File: styles/designs/carnival/parts/_preface-components.scss [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        font-size: enum('ValueSet:::NOT_NULL'),
+        padding-bottom: enum('ValueSet:::NOT_NULL'),
+        border-bottom: enum('ValueSet:::NOT_NULL'),
+        margin-bottom: enum('ValueSet:::NOT_NULL'),
+    )
+);
+
+$PrefaceH3:(
+    _name:"BodyH3",
+    _subselector: " h3",
```
Comment (tomjw64) [inline styles/designs/carnival/parts/_preface-components.scss]: Please use direct child selectors `>` only. It's better practice.

File: styles/designs/carnival/parts/_preface-components.scss [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+        font-family: enum('ValueSet:::NOT_NULL'),
+        font-weight: enum('ValueSet:::NOT_NULL'),
+        font-size: enum('ValueSet:::NOT_NULL'),
+        color: enum('ValueSet:::NOT_NULL'),
+    )
+);
+
+$PrefaceH4:(
+    _name:"BodyH4",
+    _subselector: " h4",
```
Comment (tomjw64) [inline styles/designs/carnival/parts/_preface-components.scss]: Please use direct child selectors `>` only. It's better practice.

**Comment by tomjw64** [comment]
@richelleyy Please be careful with your base branch. It looks like you based this off of your 1085 branch instead of m.m

**Pull request closed without merging**